src/backend/distributed/shared_library_init.c

```diff
@@ -329,15 +346,13 @@ RegisterCitusConfigVariables(void)
 
 	DefineCustomEnumVariable(
 		"citus.sslmode",
-		gettext_noop("SSL mode to use for connections to worker nodes."),
-		gettext_noop("When connecting to a worker node, specify whether the SSL mode"
-					 "mode for the connection is 'disable', 'allow', 'prefer' "
-					 "(the default), 'require', 'verify-ca' or 'verify-full'."),
+		gettext_noop("This GUC variable has been deprecated."),
```

Comment:
This is not a useful message. It should have also said something like "please use citus.node_conninfo"

Reply:
Opened #2243